practice_projects/#Homework_pt.py

```diff
@@ -0,0 +1,34 @@
+#Homework pt.1
+
+Friend= "\nSoftware is my friend!\n"
+print (Friend)
+
+#Homework pt.2
+
+Soft = "Software is"
+my = "my friend\n"
+print (Soft,my)
```

Comment:
I would like you to _concatenate_ the strings together instead of print each together. Create a new variable, and then concatenate the two string variables together. Then, print that variable that has the concatenation in it

Reply:
Got it. I fixed it.